hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -216,6 +216,24 @@ public static boolean isReadOnly(
     }
   }
 
+  /**
+   * Return true for the requests which does not need to call SCM via ratis.
+   *
```

Comment:
The allocate block call request, can be processed as part of StartTransaction inside Ratis's State Machine. This call is only processed on the leader and not on followers.

Reply:
Done. Thank You @mukul1987 for the suggestion, moved the newly added code to startTransaction.